lib/mini_profiler/storage/file_store.rb

```diff
@@ -17,9 +17,9 @@ def initialize(path, prefix)
         def [](key)
           begin
             data = ::File.open(path(key), "rb") { |f| f.read }
-            return Marshal.load data
```

Comment:
these offenses seem to be spotted with the new bugfixes in rubocop 0.77.0 so I had them fixed with `rubocop -a`